org-roam.el

```diff
@@ -1113,6 +1114,23 @@ Otherwise, behave as if called interactively."
 ;;;###autoload
 (defalias 'org-roam 'org-roam-buffer-toggle-display)
 
+;;;###autoload
+(defun org-roam-commit (&optional message)
+  "Return the current commit of `org-roam' repository, nil if not found.
+Interactively, or when MESSAGE is non-nil, show in the echo area."
+  (interactive)
+  (when-let ((git (executable-find "git"))
+             (library (ignore-errors (file-chase-links (locate-library "org-roam.el")))))
+    (let* ((libdir (file-name-directory library))
+           (gitdir (expand-file-name ".git" (file-name-directory libdir))))
+      (when (and (file-exists-p gitdir)
```

Comment:
It's this condition. I use Doom, so use `straight`, and with cloned git repo that resides in my local machine.
This means.... The `.git` file does not exist in this location that the previous part finds: `C:\Users\nobiot\.emacs.d\.local\straight\build\org-roam\.git`

Reply:
"this condition" is meant to be `(and file-exists-p gitdir) -- sorry if the way I did this above was not clear. I guess commenting without using the review function would have been easier. Windows seems to correctly locate where the `.el` file for the package is. The issue seems to be related to how I use Emacs and packages.

Reply:
Thanks for looking into this.

I use straight as well, but on Linux.
On *nix systems, straight's build directory symlinks *.el files back to the original repo, so we find the correct directory. IIRC, that is not the case on Windows.

Reply:
https://github.com/raxod502/straight.el/blob/a7f94876b2bf96d2595706270be6630ecc94f0d3/straight.el#L692-L695

Looks like for Windows, straight copies the source files into the build dir.